tests/scripts/deploy_admission_controller.sh

```diff
@@ -3,24 +3,54 @@
 # Sets up the environment for the admission controller webhook in the active cluster.
 set -eo pipefail
```

Comment:
Please add `-E` so that the trap can be called if an error occurs inside a function or a subshell.

Reply:
added.